packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -12,6 +12,21 @@ import SettingsContext from '../../contexts/SettingsContext';
 import { SharePostTitle } from './share';
 import { combinedClicks } from '../../lib/click';
 
+interface PostLinkProps {
+  href: string;
+  as?: string;
+  className?: string;
+  children: React.ReactNode;
+}
+
+const PostLink = ({ href, as, className, children }: PostLinkProps) => {
+  return (
+    <Link href={href} as={as}>
```

Comment:
I think the target is missing from what it used to be.
For external is should be _blank

Reply:
Same with rel